temporal-sdk/src/main/java/io/temporal/workflow/ActivityStub.java

```diff
@@ -23,14 +23,14 @@
 import java.lang.reflect.Type;
 
 /**
- * ActivityStub is used to call an activity without referencing an interface it implements. This is
```

Comment:
Nit: An activity implementation can implement multiple interfaces so I think `an` is more correct.

Reply:
thanks, changed